python/cuml/metrics/base.py

```diff
@@ -15,8 +15,12 @@
 #
 
 
+from numba import cuda
```

Comment:
```suggestion
from librmm_cffi import librmm as rmm
```

For consistency with the rest of RAPIDS and PR #1088 this would be better using RMM instead of numba directly